modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -222,3 +222,147 @@ textarea.script {
 .has-control-menu > .basecelleditor {
 	z-index: 9999 !important;
 }
+
+// LPS-132926 Migrate frontend-css-web
```

Comment:
Do we add comments like this in CSS files? 

![](https://media.giphy.com/media/TwYP72KtO8YQQ4SNgz/giphy.gif)

(I don't think so)